tasks/winbuildscripts/integrationtests.ps1

```diff
@@ -17,8 +17,8 @@ $Env:PATH="$UT_BUILD_ROOT\dev\lib;$Env:GOPATH\bin;$Env:Python3_ROOT_DIR;$Env:Pyt
 & inv -e install-tools
 & inv -e integration-tests
 $err = $LASTEXITCODE
-Write-Host Test result is $err
 if($err -ne 0){
     Write-Host -ForegroundColor Red "test failed $err"
     [Environment]::Exit($err)
 }
+Write-Host Test passed
```

Comment:
Currently, we have:
- In case of error the following trace
```
Test result is 1
Tests failed 1
```
- In case of success the following trace
```
Test result is 0
```
With my change it would be
1. In case of error
```
Tests failed 1
```
2. In case of success
```
Test passed
```
And I changed it on the various `.ps1` files to make it uniform